conda/envs/legacy/pixi.toml

```diff
@@ -26,18 +26,19 @@ libzip = "*"
 ogre = "1.10.12.1.*"
 ogre-next = "2.2.*"
 openssl = "*"
-protobuf = "*"
+protobuf = "3.14.0.*"  # closest with py3.9 ABI to jammy 3.12
 pybind11 = "*"
 python = "3.9.*"
-qt-main = "*"
+qt-main = "5.15.8"  # closets compatible with cmake using krb versions
 qwt = "*"
 ruby = "3.2.*"
 tinyxml2 = "*"
 yaml = "*"
-zeromq = "*"
+zeromq = "4.3.4"
 
 [target.win-64.dependencies]
 dlfcn-win32 = "*"
+vs2019_win-64 = "*"
```

Comment:
This is not the real compiler but the runtime libraries, we probably can leave it out.